controllers/SiteController.php

```diff
@@ -61,7 +64,19 @@ public function actions()
      */
     public function actionIndex()
     {
-        return $this->render('index');
+
+        $executors = Users::find()
+            ->joinWith('cities')
+            ->where(['is_executor' => 1])
+            ->asArray()
+            ->all();
+
+        $category_names = [];
+        foreach (Categories::find()->all() as $category) {
+            $category_names[] .= $category->name;
```

Comment:
точка в операторе присваивания тут не нужна:
```suggestion
            $category_names[] = $category->name;
```